### ISSUE TITLE
Dnd Indicator Mode? Can do it?

### ISSUE THREAD

**Issue by songispm** [kutlugsahin/vue-smooth-dnd#217]
<img width="1113" alt="image" src="https://user-images.githubusercontent.com/38745323/194370135-f0d0e1ae-bdb3-4df9-9677-ed1502f610cb.png">

example: [https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/examples-pages-layout--horizontal](https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/examples-pages-layout--horizontal)

**Comment by davestewart** [comment]
Nope. You'll need to code this by hand.

**Comment by songispm** [comment]
Even I code this by hand, but it can be achieved on the basis of vue-smooth-dnd? 

Can you give a demo when you have time? Also can add it to vue-smooth-dnd docs

**Comment by songispm** [comment]
I can try it, but please tell me, do I have to implement it by modifying the vue-smooth-dnd source code?

Or Can I implement it in my own vue vue application? Then I don't have to read the source code, just cooperate with the relevant APIs. @davestewart

**Comment by davestewart** [comment]
Yeah, pretty sure I just used mouse events and did some maths to work out where to place the divider lines.
